src/sui/themes/rtd-site/globals/site.overrides

```diff
@@ -23,8 +23,9 @@
 /*******************************
              Footer
 *******************************/
-footer {
+#footer {
```

Comment:
Perhaps `footer, #footer` is best, both should be addressable selectors

Reply:
I went for an ID as the styles applied are very specific to this `footer`.
I know it's not a common practice to have multiple footers in one page but it's perfectly acceptable to do so, should we at some point, on a different element, opt for a logical structure on which that made sense.

From [W3C](https://html.spec.whatwg.org/multipage/sections.html#the-footer-element)
> The footer element represents a footer for its nearest ancestor sectioning content or sectioning root element.

Reply:
Makes sense.

I do tend to use selectors that aren't just the element id. Though not particularly helpful in this example (`footer#footer`), in an example like `section#getting-started`, it can help provide some additional context to the element we're targeting without having to look it up in the html each time.